CONTRIBUTING.md

```diff
@@ -169,13 +169,11 @@ It is expected that you will follow the GitFlow Workflow for managing the reposi
 * On completion it should be merged **not rebased** with development
 * You may see us creating `release/<version>` branches - this is where final testing will occur.
 * Where an issue is **assigned** to somebody it means they're working on it.  Speak to them before trying to contribute.
-* Where an issue is marked as **acknowledged** then it's been agreed that it is required and can be produced.
- * We would advise against working on issues without an Acknowledged label.
- * If you want to start work on an issue, comment on the issue asking to work on it and it'll be assigned to you by a developer.
+* If you want to work on an issue. Comment on the issue asking to work on it and a member of the team will add you to the project as a Contributor. This will then allow you to work on the ticket and update it as you go.
 * Any code change **must** have an associated issue detailing (in full) why the change is being made
-* Commit messages **must** make use of the issue management commands
- * When resolving an issue via a commit `Fixes #123 - Added the user's profile picture to their profile.`
- * When addressing an issue via a commit `Restructured the user's profile page to make room for the badges in #456`
+* Commit messages **must** reference the ticket and we **recommend** using the smart commit commands available to update your JIRA issue as described [here](https://confluence.atlassian.com/bitbucket/processing-jira-software-issues-with-smart-commit-messages-298979931.html)
+ * e.g. To mark the issue for testing and to add a comment `CORE-33 #testing - #comment Added the user's profile picture to their profile.` This will update CORE-33 in JIRA with the status Testing and add the string as a comment to the issue.
```

Comment:
Building on the above, I don't believe we want to add a comment to the ticket for every commit message, or make the commit message the same as the comment. The ticket should already be linked to the branch and pull request, so a comment is actually needed, this probably isn't necessary.

Reply:
Fair enough, I will remove the reference to Smart Comments. As said above, not sure they work, they haven't for the two commits within this PR.